src/pages/index.tsx

```diff
@@ -27,37 +30,105 @@ const IndexPage: React.FC<PageProps> = ({ location }) => {
-          mainStory="Als Radschnellwege werden in Baden-Württemberg Radschnellverbindungen bezeichnet und wie folgt definiert:[1]
 
-          Gesamtstrecke mindestens fünf Kilometer
-          Potenzial von mindestens 2000 Fahrradfahrten täglich (im Durchschnitt außerorts)
-          Breite mindestens zwischen drei Metern (Richtungsverkehr) und vier Metern (Zweirichtungsverkehr)
-          in der Regel von anderen Verkehrsmitteln getrennt
-          Routen möglichst kreuzungsfrei oder mit minimalen Wartezeiten an Kreuzungen
-          Hohe Belagqualität und eine möglichst geringe Steigung
+      <TextWithImage
+        imageUrl="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/RS1_Radschnellweg_Ruhr_in_M%C3%BClheim_2.jpg/800px-RS1_Radschnellweg_Ruhr_in_M%C3%BClheim_2.jpg?uselang=de"
+        imageAlt="Zeigt den RS1 in NRW"
+        imageCredits="Wikimedia"
+        title="Deutschland wird Fahrradland"
+        caption="Hintergrund"
+      >
+        <p>
+          Radschnellverbindungen sind in Deutschland als Infrastrukturelemente
+          verhältnismäßig neu. Sie entwickeln sich seit dem verstärkten
+          Aufkommen der Elektromobilität bei Fahrrädern und dem akuten
+          Handlungsbedarf zur Erreichung von Klimaschutzzielen jedoch rasant zu
+          einem politischen und planerischen Instrument mit hohem Potential und
+          großer Akzeptanz. Die hohe Zahl von Machbarkeitsstudien und
+          Vorplanungen in Kommunen in Deutschland bestätigt dies. Adäquate
+          Online-Beteiligungsformate für die besonderen Anforderungen bei der
+          Planung von Radschnellverbindungen sind bisher kaum vorhanden. Die
+          digitale Konsultation und Beteiligung von Bürger:innen wird bisher nur
+          eingeschränkt, meist mittels allgemeiner Online-Beteiligungs-Tools
+          durchgeführt.
+        </p>
+      </TextWithImage>
+      <TextWithImage
+        imageUrl="https://images.unsplash.com/photo-1541687664971-639c2f8b63f3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80"
+        imageAlt="TestAlt"
+        imageCredits=""
+        title="RSV-Baukasten"
+        caption="Projektansatz"
+      >
+        <p>
+          Mit dem RSV-Baukasten entsteht ein modulares System zur Durchführung
+          von Online-Beteiligungsformaten und Bürgerinformation zu
+          Radschnellverbindungen, das Bedarfe und Wissen konstruktiv in
+          Planungsprozesse einfließen lässt. Grundlage ist ein dynamisches
+          Online-Kartenformat das Informationen zu den geplanten
+          Radschnellverbindungen übersichtlich aufbereitet und
+          Beteiligungsinstrumente daran anbindet.
+        </p>
+        <b>Zentrale Ziele des geplanten RSV-Baukastens:</b>
+        <ul className="list-inside list-disc">
```

Comment:
Ich versuche bei TW immer "list-inside" zu vermeiden, weil es bei längeren Texten immer sehr schlecht aussieht, weil die zweite Zeile unter den Bullet steht. https://tailwindcss.com/docs/list-style-position#hover-focus-and-other-states

Es müsste IMO eher …
```suggestion
        <ul className="list-disc ml-5">
```
… sein (ich meine list-outside ist default).

Habe sogar nen PR dazu gemacht mal :) https://github.com/tailwindlabs/tailwindcss.com/pull/951/files